Aliases/azion

```diff
@@ -0,0 +1 @@
+../Formula/a/azion-cli.rb
```

Comment:
let's not do this unless it is absolutely necessary

Reply:
Most of our users now the product by the name "azion". It would be more intuitive to be able to install it by this name, that's why I've added the Alias.

Reply:
then we not rename the formula to azion in the first place?

Reply:
also we do have misspell hint builtin with `brew` (suggesting the right formula name)

Reply:
Sure, I could rename the formula to Azion :)

Reply:
I thought that was not a possibility

Reply:
There's a `formula_renames.json` file in the root of the repo. Just add this formula in this PR.

Reply:
Done